Update threat-intelligence-integration.md (#84012)
Azure/azure-content: articles/sentinel/threat-intelligence-integration.md

- [Learn more about STIX and TAXII @ThreatConnect](https://threatconnect.com/stix-taxii/)
- [TAXII Services documentation @ThreatConnect](https://docs.threatconnect.com/en/latest/rest_api/v2/taxii/taxii.html)

### Sectrio

- [Learn more about Sectrio integration](https://sectrio.com/threat-intelligence/)

> - [Step by step process of how to bring in Sectrio feed into Microsoft Sentinel](https://techcommunity.microsoft.com/t5/microsoft-sentinel-blog/microsoft-sentinel-bring-threat-intelligence-from-sectrio-using/ba-p/2964648)

Reviewer: ```suggestion
- [Step by step process for integrating Sectrio's TI feed into Microsoft Sentinel](https://techcommunity.microsoft.com/t5/microsoft-sentinel-blog/microsoft-sentinel-bring-threat-intelligence-from-sectrio-using/ba-p/2964648)
```
